apps/src/netsim/NetSimPanel.html.ejs

```diff
@@ -2,7 +2,9 @@
      class="netsim-panel <%= className %>">
   <h1>
     <span class="minimizer">
+      <% if (canMinimize) { %>
       <i class="fa fa-minus-square"></i>
```

Comment:
My convention has been yes; not sure from our style guide what's correct for ejs though.